playground/TestShop/MyFrontend/Program.cs

```diff
@@ -8,10 +8,10 @@
 
 builder.Services.AddHttpForwarderWithServiceDiscovery();
 
-builder.Services.AddHttpClient<CatalogServiceClient>(c => c.BaseAddress = new("http://catalogservice"));
+builder.Services.AddHttpClient<CatalogServiceClient>(c => c.BaseAddress = new("https+http://catalogservice"));
```

Comment:
This is the next syntax for multi-scheme resolution being added to service discovery by @ReubenBond as part of #381